### DISCUSSION TITLE
How to put a warning inside code

### DISCUSSION THREAD

**Discussion by DeDiamondPro** [community/community#25672] [Actions]
<p>Hi,<br>
I’m trying to write my own action in javascript and a bit ago I saw an action create a warning in the source code which I would also like to do but after looking around a bit I couldn’t find anything telling me how to do it.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/6/761c110297eb767e39901ff57695e5b50622873d.png" data-download-href="/uploads/short-url/gQQtmv9dxRazSI3Jdt6AYyGXIx7.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/67508414/181092451-1170bae7-ebe5-479c-bc90-d54aaeb65620.png" alt="image" data-base62-sha1="gQQtmv9dxRazSI3Jdt6AYyGXIx7" width="690" height="71" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/761c110297eb767e39901ff57695e5b50622873d_2_690x71.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/761c110297eb767e39901ff57695e5b50622873d_2_1035x106.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/6/761c110297eb767e39901ff57695e5b50622873d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/761c110297eb767e39901ff57695e5b50622873d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1228×127 10.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
(example of what I mean)
<p>Thanks in advance!</p>

**Comment by kingthorin** [comment, marked as answer]
<p>I believe those are referred to as “annotations”</p>

**Comment by DeDiamondPro** [comment]
<p>Thanks after googling around a bit I found some things that got me on my way</p>

**Comment by kingthorin** [comment]
<p>Great, glad I could help :slightly_smiling_face:</p>
